hbase/src/main/java/org/apache/zeppelin/hbase/HbaseInterpreter.java

```diff
@@ -143,30 +156,26 @@ public Scheduler getScheduler() {
         HbaseInterpreter.class.getName() + this.hashCode());
   }
 
-  @Override
-  public List<InterpreterCompletion> completion(String buf, int cursor,
-      InterpreterContext interpreterContext) {
-    return null;
+  private void stopProcess(String paragraphId) {
+    if (runningProcesses.containsKey(paragraphId)) {
```

Comment:
The deletion of values can also be written more elegantly here. The code currently accesses the map three times.
1) containsKey
2) get
3) remove

Reply:
Hi @Reamer ,
Thanks. I will get it done.